SparkPost/2web2ui
# FE-1285 - Add new ESLint rule warning when the user imports moment

[FE-1285](https://sparkpost.atlassian.net/browse/FE-1285)

### What Changed

- Adds new rule that warns when importing `moment` with a custom warning message

### How To Test

- Open a JS module that is importing `moment`, verify that an ESLint warning occurs
- Does the warning message make sense to you?

### To Do
